save-common/src/jsMain/kotlin/com/saveourtool/save/core/utils/ProcessBuilder.kt

```diff
@@ -1,22 +1,17 @@
-@file:Suppress(
-    "MISSING_KDOC_TOP_LEVEL",
-    "MISSING_KDOC_CLASS_ELEMENTS",
-    "KDOC_NO_EMPTY_TAGS",
-    "MISSING_KDOC_ON_FUNCTION",
-    "FILE_NAME_MATCH_CLASS",
-    "MatchingDeclarationName",
-)
+/**
+ * Implementation of [ProcessBuilderInternal] for JS
+ */
 
 package com.saveourtool.save.core.utils
 
 import okio.Path
 
-actual class ProcessBuilderInternal actual constructor(
```

Comment:
what's the point to change classes into methods?

Reply:
Kotlin suggests such pattern: https://www.jetbrains.com/help/kotlin-multiplatform-dev/multiplatform-connect-to-apis.html#interfaces-in-common-code

Also, actual\expect classes are in Beta (gradle warns about it): https://youtrack.jetbrains.com/issue/KT-61573/Emit-the-compilation-warning-on-expect-actual-classes.-The-warning-must-mention-that-expect-actual-classes-are-in-Beta